src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({
+  todo: {
+    id, title, completed, user,
+  },
+}) => (
+  <article
+    data-id={id}
+    className={classNames('TodoInfo',
+      { 'TodoInfo--completed': completed })}
+  >
+    <h2 className="TodoInfo__title">{title}</h2>
+    <UserInfo user={user} />
```

Comment:
you can check the user is not null before rendering the component
```suggestion
    {user && <UserInfo user={user} />}
```